miner/README.org

```diff
@@ -175,6 +175,66 @@ You may only be interested in warning or error messages. If so, use the
   2019-09-16 16:58:23.646547: [error] Failed to fetch work! Is the Node down?
 #+end_example
 
+*** Specifying Mulitple Nodes
+
+You can specify multiple nodes to mine from the command line.
+*However*, what this actually means needs to be clarified. Consider
+this example:
+
+#+BEGIN_SRC bash
+  ./chainweb-miner cpu --node=us-e3.chainweb.com:443
+  --node=us-e2.chainweb.com:443 --miner-account example
+  --miner-key example --miner-pred keys-all --chain 0
```

Comment:
```
./chainweb-miner cpu --node=us-e3.chainweb.com:443
  --node=us-e2.chainweb.com:443
```
I'd rather have this omit the rest of the arguments and fail to prevent users from just copy-ing and pasting with an incorrect key